abb_hardware_interface/CMakeLists.txt

```diff
@@ -29,13 +32,24 @@ set(THIS_PACKAGE_INCLUDE_DEPENDS
 # Add Poco and TinyXML2 to dependencies
 find_package(Poco REQUIRED Foundation Net NetSSL XML)
 
-# find dependencies
+# Find dependencies
 find_package(ament_cmake REQUIRED)
 
 foreach(Dependency IN ITEMS ${THIS_PACKAGE_INCLUDE_DEPENDS})
   find_package(${Dependency} REQUIRED)
 endforeach()
 
+###########
```

Comment:
I guess it would be nicer if the length of this matches the next row. Same afterwards.